koocook_core/views/mixins.py

```diff
@@ -123,6 +128,7 @@ def process_equipment(self, form):
     # Messy
     def form_valid(self, form):
         response = super().form_valid(form)
+        LOGGER.info(f"{self.get_visitor_name()} has {self.ACTION}d the recipe named {form.instance.name} #{form.instance.id}")
```

Comment:
Adding quotes around the name of the recipe would make the log more readable (reader knows what part is the name and what is not).

Reply:
Also, instead of adding 'd' to each actions, it might be better to define the words directly, and use that.

Reply:
I do not want to interpolate much into generic views, and the fix could be done later. This branch is simply a frame for other branches to add logging on top of the features.